### ISSUE TITLE
git: I can't identify the type of change in the list of modified files

### ISSUE THREAD

**Issue by jvesouza** [microsoft/vscode#94464]
In the list of modified files presented when we press ctrl + g, we can have modified files, added files and files outside the scope of the folder that was opened in VSCode.
I use orca as a screen reader and I can't identify what file is modified, added and out of scope.

Would it be possible to display the files separated by the type of modification?
Today the display is already done separating the modified files and the staged files.
Would it be possible to use a similar approach?
CC @isidorn

**Comment by joaomoreno** [comment]
@isidorn Can you provide direction here?

**Comment by isidorn** [comment]
@jvesouza thanks for filling the issue
While making the presentation structure more fine grained makes sense for your use case, however I think it would introduce unneeded complexity. We try to make the Git view a nice minimalistic represtantion of the git status of your workspace. And due to that I suggest that the `ariaLabel` of each item should have the item status at the end.
So if you have `hello.txt` which is staged, `test.txt` which is modified and `tmp.txt` which is added I would expect the following area labels:
"hello.txt staged"
"test.txt modified"
"tmp.txt added"

Having the state at the end makes it easier for screen readers users to simply skip it.
@joaomoreno @jvesouza let me know what you think about my suggestion

**Comment by joaomoreno** [comment]
Will be fixed when https://github.com/microsoft/vscode/issues/92342 is closed.

**Comment by jvesouza** [comment]
Sounds good to me.

**Comment by jvesouza** [comment]
I noticed that now when we browse the list of modified files, the status of the file, modified, untracked, etc. is displayed. However, orca does not announce the patch to which the file belongs, which can be a problem in repositories in which we have files with the same name in different folders.
The java project I'm working on has two files called config.propertes and both have been modified. I just wanted to commit one of them and had a hard time finding the right file.

**Comment by isidorn** [comment]
@jvesouza good feedback and makes sense. We can also announce the path of the file, but would you expect first to hear "modified" or the path? Would this make the screen reader too chatty? Feels similar to this https://github.com/microsoft/vscode/issues/93657
So if we change one we should change the other.

So I suggest to use the following aria label NAME RELATIVE_PATH STATE

Reopening so we also get @joaomoreno feedback on this

**Comment by jvesouza** [comment]
Modified first.

**Comment by isidorn** [comment]
Reopening, since we should fix this small nitpick.
I think first the name, than modified than path.
Currently you first announce modified, then name than path.

I think @jvesouza would agree. But let's reoepen and move to May.

**Comment by joaomoreno** [comment]
From @jvesouza's last comment, I understood `Modified` would come first...

**Comment by isidorn** [comment]
@joaomoreno I think he meant before RELATIVE_PATH. It should not be bfore the NAME, the NAME is the most important imho

**Comment by jvesouza** [comment]
I personally liked the way it was implemented, modified before the name. If it depends only on my opinion, the implementation can remain as it is.
Ideally, other users should give their opinion.

**Comment by alex19EP** [comment]
I agree with @isidorn name of the most important then status, and then path.

**Comment by ndarilek** [comment]
I don't care about status/path, but definitely name first please. Way more often than not, I'm just changing files that exist. So hearing "Modified" before every file definitely slows me down, even if only slightly.

I guess if I had to have an opinion, I'd say "Name, path, status." The name is most important. Next I need the path, because unfortunately I have many similarly-named files. Finally I need the status. Usually I can infer the status given the name and path, but if I can't, then having it last gives me the most important information I need. I know what I need to do for a specific file given its name and path.

Note that I fairly quickly set:

```
    "git.smartCommitChanges": "tracked",
    "git.enableSmartCommit": true,
    "git.untrackedChanges": "separate",
```

This separates out untracked files, and doesn't stage things I don't want staged. Unfortunately, at one point I had some non-gitignored secrets that got committed. This makes committing something I have to explicitly plan, which is as it should be IMO.

**Comment by webczat** [comment]
I am wondering why the name and path are not one element tbh. Like first time it confused me a little. Although I could get used to that.

**Comment by webczat** [comment]
as for the order, somehow for me path belongs more to the filename so it should be before the modified.

**Comment by akash07k** [comment]
Agreed
> I don't care about status/path, but definitely name first please. Way more often than not, I'm just changing files that exist. So hearing "Modified" before every file definitely slows me down, even if only slightly.
> 
> I guess if I had to have an opinion, I'd say "Name, path, status." The name is most important. Next I need the path, because unfortunately I have many similarly-named files. Finally I need the status. Usually I can infer the status given the name and path, but if I can't, then having it last gives me the most important information I need. I know what I need to do for a specific file given its name and path.
> 
> Note that I fairly quickly set:
> 
> ```
>     "git.smartCommitChanges": "tracked",
>     "git.enableSmartCommit": true,
>     "git.untrackedChanges": "separate",
> ```
> 
> This separates out untracked files, and doesn't stage things I don't want staged. Unfortunately, at one point I had some non-gitignored secrets that got committed. This makes committing something I have to explicitly plan, which is as it should be IMO.

**Comment by isidorn** [comment]
@joaomoreno pushed a fix for this, hope that is ok. It was very simple.
